config/laratrust_seeder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+return [
+    /**
+     * Control if the seeder should create a user per role while seeding the data.
+     */
+    'create_users' => false,
+
+    /**
+     * Control if all the laratrust tables should be truncated before running the seeder.
+     */
+    'truncate_tables' => true,
+
+    'roles_structure' => [
+        'superadministrator' => [
+            'calendar' => 'c,r,u,d',
+            'flow-board' => 'c,r,u,d',
+            'patient' => 'c,r,u,d',
+            'fees' => 'c,r,u,d',
+            'inventory' => 'c,r,u,d',
+            'procedures' => 'c,r,u,d',
+            'administration' => 'c,r,u,d',
+            'reports' => 'c,r,u,d',
+            'misc' => 'c,r,u,d',
+            'popups' => 'c,r,u,d',
+            'qa-measures' => 'c,r,u,d',
+            'help' => 'c,r,u,d',
+        ],
+        'administrators' => [
+            'calendar' => 'c,r,u,d',
+            'flow-board' => 'c,r,u,d',
+            'patient' => 'c,r,u,d',
+            'fees' => 'c,r,u,d',
+            'inventory' => 'c,r,u,d',
+            'procedures' => 'c,r,u,d',
+            'administration' => 'c,r',
+            'reports' => 'c,r,u,d',
+            'misc' => 'c,r,u,d',
+            'popups' => 'c,r,u,d',
+            'qa-measures' => 'c,r,u,d',
+            'help' => 'r',
+        ],
+        'lh-user' => [
```

Comment:
Let's call this role `users`

Reply:
Ok, I have updated it in the recent commit.